client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -146,6 +146,17 @@ class SharingServiceDescription extends Component {
 					comment: 'Get slack notifications on new P2 posts.',
 				} );
 			},
+			p2_github: function () {
+				if ( this.props.numberOfConnections > 0 ) {
+					return this.props.translate( 'Workspace connected to Github.', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Workspace connected to Slack.' )` *ES Score: 7*